bot/exts/evergreen/quack.py

```diff
@@ -0,0 +1,65 @@
+from typing import Optional
+
+import discord
+from discord.ext import commands
+
+from bot.bot import Bot
+from bot.constants import Colours
+
+
+class Quack(commands.Cog):
+    """Cog for the .quack command."""
+
+    def __init__(self, bot: Bot) -> None:
+        self.bot = bot
+
+    @commands.command()
+    async def quack(self, ctx: commands.Context, seed: Optional[str], ducktype: Optional[str] = None) -> None:
```

Comment:
This current signature doesn't let the user generate a manduck without specifying a seed from what I can tell. Maybe we could swap the ducktype and seed around? Any other ideas for how to fix this?

Also wouldn't it make sense for `ducktype` to default as `duck`, not `None`